File: sprint1/kotlin/lesson2code1.kt
fun findElement(numbers: List<Int>, x: Int): Int {
    for (i in numbers.indices) { // проходим по всем элементам массива
        if (numbers[i] == x) { // сравниваем их с иксом
            return i  // если нашли - возвращаем индекс
        }
    }
    return -1  // если не нашли - возвращаем -1
}

File: sprint1/C#/lesson2code1.cs
public static int FindElement(int[] numbers, int x)
{
    for (int i = 0; i < numbers.Length; i++) // проходим по всем элементам массива
    {
        if (numbers[i] == x) // сравниваем их с иксом
        {
            return i;  // если нашли - возвращаем индекс
        }
    }
    return -1;  // если не нашли - возвращаем -1
}
